vscode/src/jdkDownloader/downloader.ts

```diff
@@ -0,0 +1,263 @@
+import { commands, OpenDialogOptions, OutputChannel, ProgressLocation, window, workspace } from "vscode";
```

Comment:
can we rename it to downloader.action.ts -> as this would be absrtracting vcode actions